### ISSUE TITLE
Widget "Open page" does not show title if dashboard hidden from side panel

### ISSUE THREAD

**Issue by Alex9779** [home-assistant/iOS#1913]
**iOS device model, version and app version**

Model Name: iPhone 12
Software Version: 15.1
App version: 2021.11.2

**Home Assistant Core Version**
2021.11.2

**Describe the bug**
Additional dashboards have the option to hide them from the side panel. If you use the iOS widget to open a specific page in the app you do not see the title of the dashboard in the selection and also not in the widget if the dashboard is set to be hidden from the side panel, it just shows "lovelace". If the dashboard is visible in the side panel the title is show as expected.

**To Reproduce**
Create a dashboard which is hidden from the side panel, add a widget in iOS to "open page" and search for the new dashboard.

**Expected behavior**
Dashboard titles shown in the list and the widget also if dashboard not visible in side panel.

**Screenshots**

Dashboard visible:
<img width="425" alt="Dashboard-visible" src="https://user-images.githubusercontent.com/6394581/141321325-4d16c67b-f680-401e-a83e-1cb17c28f7c5.png">
<img width="425" alt="Dashboard-visible"  src="https://user-images.githubusercontent.com/6394581/141321431-089234bd-8ce5-4d05-9311-abec911ef752.png">

Dashboard invisible:
<img width="425" alt="Dashboard-invisible" src="https://user-images.githubusercontent.com/6394581/141321351-8cf6e7dc-18c9-47e5-9886-529402eae221.png">
<img width="425" alt="Dashboard-visible" src="https://user-images.githubusercontent.com/6394581/141321528-52811c73-f838-4df3-85e1-7c4020a99f96.png">

**Additional context**

**Comment by zacwest** [comment]
I believe this is going to require core-side updates to fix this issue -- the data is in this format, and the key I thought was coming down (for us to hide these panels) is not, so there's no easy way to drop it right now.

**Comment by Alex9779** [comment]
Yeah sorry :D just played with it, thought it would be nice to have a shortcut to my maintenance dashboard and not to overload my sidebar, or to have a shortcut to a special dashboard with controls I just need when I am outside and need a quick access...
But without a proper title it is not so nice, I could change the color of the widgets but all would be named "Lovelace" so...
Just tell me if I have to take some action.
It is not that kind of life changing feature, just would be nice if it would be possible to be resolved sometime...

**Comment by zacwest** [comment]
If what you mean is that you'd like to not see them for now, you can hide certain pages from the widget by long-pressing it and choosing "Edit Widget".

**Comment by Alex9779** [comment]
That is not exactly what I meant. I wasn't talking about the widget with a list of dashboards, but thanks for pointing that out, I didn't notice that this is also possible and available. Though this has the same issue and just shows "lovelace" for hidden dashboards.
It is just the other way, I want to have hidden dashboards in the list or just that square widget with just one dashboard link to a hidden dashboard.
What I also meant by "action I have to take" is if I should create some issue in the core, though I am not really aware of the technical background you described and what is needed in the core so you can fix this.

**Comment by zacwest** [comment]
You don't need to file anything elsewhere, I will take care of it.

**Comment by MJWMJW2** [comment]
I am glad to find this reporting the same issue I was about to mention.  Looks like it didn't receive any attention as still an issue some years later.

**Comment by JuergenBuchner** [comment]
I have the same problem
